### PR TITLE
edsl.md: add uint16 ABI calldata type

### PR DESCRIPTION
**Pull request by livnev**
Not sure what the policy on ABI type inclusion is, but this is a solidity type that came up in a spec we are working on.

### CODE REVIEW

**Comment by ehildenb** [comment]
@livnev you'll have to sign the CLA (click the "Details" button next to the red X).

**Comment by livnev** [comment]
@ehildenb done

File: data.md [hunk 1]
```diff
@@ -81,7 +83,8 @@ These can be used for pattern-matching on the LHS of rules as well (`macro` attr
 
     rule minUInt8        =>  0                                                                              [macro]
     rule maxUInt8        =>  255                                                                            [macro]
-    rule minUInt48       =>  0                                                                              [macro]
+    rule minUInt16       =>  0                                                                              [macro]
+    rule maxUInt16       =>  65535                                                                          [macro]  /*   2^16 -  1  */    rule minUInt48       =>  0                                                                              [macro]
```
Comment (ehildenb) [inline data.md]: Some sort of line-spacing typo here.

Comment (livnev) [reply data.md]: Fixed.

**Pull request merged**